components/dropzone/index.css

```diff
@@ -144,55 +84,97 @@
 	}
 
 	&.is-filled.is-dragged {
-		--mod-drop-zone-content-display: flex;
+		background-position: center;
+		block-size: var(--mod-drop-zone-content-height);
+		background-blend-mode: multiply;
+
+		.spectrum-DropZone-content {
+			block-size: 100%;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+		}
+
+		.spectrum-DropZone-actions {
+			margin-block-start: 0;
+		}
 	}
 
 	&:focus-visible {
-		--mod-drop-zone-border-style: solid;
 		--spectrum-drop-zone-border-color: var(--highcontrast-drop-zone-border-color-hover, var(--mod-drop-zone-border-color-hover, var(--spectrum-drop-zone-border-color-hover)));
 		outline: 0;
 	}
 }
 
-.spectrum-DropZone-content {
-	display: var(--mod-drop-zone-content-display, none);
+.spectrum-DropZone:has(.spectrum-DropZone-stroke) {
+	border: none;
+}
```

Comment:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExd2ZraGpkbnJmMjVuejFxOHhpdDBvYzVpYnAyNHg1Z2V0OWR5cGhxYSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/5GoVLqeAOo6PK/giphy.gif)

Aw this fallback works so nicely 😍